pkgs/development/compilers/elm/packages/elm-i18next-gen.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildNpmPackage, fetchFromGitHub }:
+
+buildNpmPackage rec {
+  pname = "elm-i18next-gen";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "yonigibbs";
+    repo = pname;
```

Comment:
```suggestion
    repo = "elm-i18next-gen";
```

Reply:
Why does this need to be changed?

Reply:
https://github.com/NixOS/nixpkgs/issues/277994